src/app.html

```diff
@@ -1,12 +1,12 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%sveltekit.assets%/favicon.png" />
-    <meta name="viewport" content="width=device-width" />
-    %sveltekit.head%
-  </head>
-  <body data-sveltekit-preload-data="hover">
-    <div style="display: contents">%sveltekit.body%</div>
-  </body>
+<!DOCTYPE html >
+<html lang="en" class="dark">
```

Comment:
Ich würde den Dark-Mode nicht hardcoden, sondern anhand des User-Agents automatisch bestimmen. Später kann man es in den Einstellungen vom Benutzer per `Lightswitch` von `Skeleton` überschreiben lassen.

Reply:
Macht Sinn